manifests/crab-darwinia.yaml

```diff
@@ -1,27 +0,0 @@
-manifestVersion: subsquid.io/v0.1
-name: gs-main-crab
-version: 1
-description: |-
-  Giant Squid Main for Darwinia Crab Network
-build:
-deploy:
-  addons:
-    postgres:
-  processor:
-    env:
-      CHAIN: crab
-    cmd: ["node", "lib/main"]
-  api:
-    cmd:
-      [
-        "npx",
-        "squid-graphql-server",
-        "--dumb-cache",
-        "in-memory",
-        "--dumb-cache-ttl",
-        "1000",
-        "--dumb-cache-size",
-        "100",
-        "--dumb-cache-max-age",
-        "1000",
-      ]
```

Comment:
The only file to be deleted instead of modified?

Reply:
It's Renamed to darwinia-crab.yaml as that is the right name, in the Pr it shows 1 deleted and 1 new file